module/src/main/java/com/annimon/tgbotsmodule/services/CommonAbsSender.java

```diff
@@ -582,16 +597,22 @@ public void onException(BotApiMethod<T> method, Exception exception) {
         });
     }
 
+    private <T extends Serializable, Method extends BotApiMethod<T>> void preprocessMethod(@NotNull Method method) {
+        var preprocessor = preprocessors.get(method.getMethod());
+        if (preprocessor != null)
+            preprocessor.accept(method);
+    }
+
     private interface ResultSupplier<T> {
         T get() throws TelegramApiException;
     }
 
     /**
      * Handles exceptions for {@code call} methods.
-     *
+     * <p>
      * By default logs exception as an error.
      *
-     * @param ex  Exception
+     * @param ex Exception
```

Comment:
JavaDoc guidelines require at least two spaces between argument name and description.
But this rule doesn't apply to type parameters:

```
@param name  two spaces
@param <T> one space
@return one space
```